.gitignore

```diff
@@ -9,6 +9,7 @@ workflow/rendered/
 */bin
 tanagra-indexing.html
 diffReleases.txt
+ui/src/tanagra-underlay/
```

Comment:
Move this to `ui/.gitignore`.

Reply:
Still failed. Just going to fix the lint errors.